WordPress/src/test/java/org/wordpress/android/viewmodel/SpeedDialViewModelTest.kt

```diff
@@ -0,0 +1,68 @@
+package org.wordpress.android.viewmodel
+
+import androidx.arch.core.executor.testing.InstantTaskExecutorRule
+import androidx.lifecycle.Observer
+import kotlinx.coroutines.InternalCoroutinesApi
+import kotlinx.coroutines.runBlocking
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.Mock
+import org.mockito.junit.MockitoJUnitRunner
+import org.wordpress.android.R
+import org.wordpress.android.TEST_DISPATCHER
+import org.wordpress.android.ui.main.SpeedDialAction
+import org.wordpress.android.ui.main.SpeedDialState.CLOSED
+import org.wordpress.android.ui.main.SpeedDialState.HIDDEN
+import org.wordpress.android.ui.main.SpeedDialUiState
+import org.wordpress.android.ui.main.SpeedDialViewModel
+
+@InternalCoroutinesApi
+@RunWith(MockitoJUnitRunner::class)
+class SpeedDialViewModelTest {
+    @Rule
+    @JvmField val rule = InstantTaskExecutorRule()
+
+    @Mock private lateinit var uiStateObserver: Observer<SpeedDialUiState>
+
+    private lateinit var viewModel: SpeedDialViewModel
+
+    @Before
+    fun setUp() {
+        viewModel = SpeedDialViewModel(TEST_DISPATCHER)
+        viewModel.uiState.observeForever(uiStateObserver)
```

Comment:
👍! Actually I do not know if I'm missing something but it seems we can completely remove the observer since we are not using it and the framework is not complaining and doing it's job! Changed here 368564eb65bf326986aa5017d6a852f078d8d9a4